src/pages/activities/[page].tsx

```diff
@@ -0,0 +1,37 @@
+import { getActivityByName, getAllActivities } from "../../libs/activitiesAPI";
+
+const Page = (page: JSON) => {
+  return JSON.stringify(page);
+};
```

Comment:
```suggestion
import React from "react"
const Page: React.FC = ({ page }) => {
  return <pre>JSON.stringify(page)</pre>;
};
```

In reality the default export should be a react component, this is a good example that rishabh can use and replace with the UI.